### PR TITLE
Package cull

### PR DESCRIPTION
**Pull request by kjhenner**

### CODE REVIEW

File: templates/packages.erb [hunk 1]
```diff
@@ -13,20 +10,14 @@ httpd-itk
...
-mg
 nano
 net-tools
-nginx
 ntp
 ntpdate
 openssh
 openssh-server
 openssl
-php
```
Comment (carthik) [inline templates/packages.erb]: This, and others, will be needed for the classroom training labs. Are the list of packages for the training/puppetfactory VM elsewhere?

Comment (carthik) [reply templates/packages.erb]: Nevermind, I see the unless.

**Comment by carthik** [comment]
I don't see any obvious issue. Perhaps @binford2k should review this as well.

**Comment by kjhenner** [comment]
@carthik This is against the legacy branch, which is currently only used for the Learning VM build. If @binford2k has suggestions, I'd be interested to hear them, but I don't believe his input is necessary if your concern is related to the impact on classroom VMs.

**Comment by carthik** [comment]
Thanks @kjhenner. I spoke with @binford2k

**Pull request merged**